src/utils/layout.js

```diff
@@ -135,6 +140,7 @@ export function getSectionComponent(section, maxWidth) {
           customClass={isPageBreakSection(section) ? 'page-break-section' : ''}
           forceRangeMessage={section.layout.forceRangeMessage}
           markdownArtifactsServerAddress={section.markdownArtifactsServerAddress}
+          isBordered={shouldSectionBeNotBordered(section)}
```

Comment:
`isBordered` prop value received from a function named - shouldSectionBe**NotBordered**.
Please improve the naming.

If the function `shouldSectionBeNotBordered` return's `true`, it means that it should not be bordered?

Reply:
ummm.. it's a bit confusing, the function referees to the parent section. i will rename that

Reply:
done.